README.md

```diff
@@ -420,12 +486,17 @@ The Flashlight CMake build accepts the following build options (prefixed with `-
 
 ### Building Your Own Project with Flashlight
 Flashlight is most-easily linked to using CMake. Flashlight exports the following CMake targets when installed:
-- `flashlight::fl-libraries` — contains flashlight libraries headers and symbols.
+- `flashlight::fl_lib_common` — contains flashlight libraries for common headers and symbols used throughout flashlight.
+- `flashlight::fl_lib_set` — contains flashlight libraries for headers and symbols pertaining to sets.
+- `flashlight::fl_lib_sequence` — contains flashlight libraries for headers and symbols pertaining to sequences.
+- `flashlight::fl_lib_audio` — contains flashlight libraries for headers and symbols pertaining to audio.
+- `flashlight::fl_lib_text` — contains flashlight libraries for headers and symbols pertaining to text.
 - `flashlight::flashlight` — contains flashlight libraries as well as the flashlight core autograd and neural network library.
```

Comment:
This is no longer correct (re `flashlight::flashlight`) - can we update it?

Reply:
We still have the `flashlight` library which is core. We no longer have fl-libraries.